components/authentication/AuthCheckBox/AuthCheckBox.tsx

```diff
@@ -12,7 +12,7 @@ const AuthCheckBox: React.FC<PropsType> = (props) => {
         type='checkbox'
         name={props.name}
         defaultValue={props.value}
-        className='w-4 h-4'
+        className='form-checkbox w-4 h-4 text-custom-blue border-custom-gray rounded-[4px] duration-200 !ring-offset-0 !ring-0'
```

Comment:
hmm, didn't notice thisa before, do not group files like this, if u are using something in multiple pages just place them in shared, group them as shared or as page name, i do not see page called authentication, move them out